src/containers/Cinema/Cinema.tsx

```diff
@@ -130,9 +138,10 @@ const Cinema: React.FC<Props> = ({ item, onPlay, onPause, onComplete, onUserActi
       if (currentItem && currentItem.mediaid === item.mediaid) {
         return;
       }
-
       // load new item
-      playerRef.current.setConfig({ playlist: [deepCopy(item)], autostart: true });
+      playerRef.current.load([deepCopy(item)]);
```

Comment:
I guess this is an attempt to fix an issue where the event listeners are using a stale state. 

We've also been experiencing this issue and created a different fix for this. 

The current fix will only work when a new playlist is loaded in the player but doesn't work when a different state value changes that are used in the event callbacks.

I.M.O. we should try to prevent (un)binding event listeners on every state update (we've found other issues using this method). For this reason, I suggest using the `useEventCallback` hook instead. By using this, you'll only need to bind the event listeners on player initialization and unbind on unmount (probably redundant since the player instance is destroyed on unmount).

e.g.

```ts
  const handlePlay = useEventCallback(onPlay);

  useEffect(() => {
    playerRef.current.setup({
      playlist: [deepCopy(item)],
      ...
      repeat: false,
    });

    playerRef.current.on('play', handlePlay);
    // rest
  }, []);

  useEffect(() => {
    return () => {
      if (!playerRef.current) return;
      
      playerRef.current.off('play', handlePlay);
      // rest
    };
  }, []);
```